### PR TITLE
Disable building snippets for more libraries

### PR DESCRIPTION
**Pull request by heaths**
Relates to #33969

### CODE REVIEW

**Review by jsquire — APPROVED** [review]

File: sdk/tables/ci.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: tables
+    BuildSnippets: false
```
Comment (christothes) [inline sdk/tables/ci.yml:28]: You can revert this - I fixed the snippets in https://github.com/Azure/azure-sdk-for-net/pull/33998

**Review by m-nash — APPROVED** [review]

File: sdk/tables/ci.yml (hunk 1 above)
Comment (heaths) [reply sdk/tables/ci.yml:28]: As long as the validation on this PR takes, I'd rather wait till after so it just touches the _sdk/tables_ directory. I'll leave this unresolved for now so I don't forget while the 1 failed run on MacOS re-runs.

**Comment by heaths** [comment]
@christothes discovered the source of the breaks on macOS (or maybe just because that was the only agent that exercised those tests) and is prepping a separate PR now. As such, overriding this one since every other test passed.

**Pull request merged**